cmd/nbe/parse.go

```diff
@@ -65,6 +65,7 @@ var (
 		DotNet:    "Main.cs",
 		DotNet2:   "Main.cs",
 		Elixir:    "main.exs",
+		C:         "main.c",
```

Comment:
1/5 leave it there, helps with debugging locally but not enough to make it an `nbe` flag